react-class-components/src/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+function CustomButton(props) {
```

Comment:
so this is a function component, you want to define a class component for this exercise

Reply:
refactored